lib/handlers/app/handlers/getFullTheme.js

```diff
@@ -0,0 +1,30 @@
+var forms = require('fh-forms');
+var logger = require('../../../util/logger').getLogger();
+
+/**
+ * Get A Theme Associated With A Project
+ * @param {object}   req    Request Object
+ * @param {object}   res    Response Object
+ * @param {function} next   Middleware Next Function
+ */
+module.exports = function getFullTheme(req, res, next) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)